libbeat/tests/system/beat/compose.py

```diff
@@ -215,3 +215,17 @@ def find_compose_path(cls):
             class_dir, current = os.path.split(class_dir)
             if current == '':  # We have reached root
                 raise Exception("failed to find a docker-compose.yml file")
+
+    @classmethod
+    def get_service_log(cls, service):
+        container = cls.compose_project().containers(service_names=[service])[0]
+        return container.logs()
+
+    @classmethod
+    def service_log_contains(cls, service, msg):
```

Comment:
I have deliberately chosen to keep the implementation of this function simple — without regex matching or case ignoring. We can add those features as we need them, IMO.